src/compiler/sys.ts

```diff
@@ -1578,7 +1578,8 @@ export let sys: System = (() => {
             disableCPUProfiler,
             cpuProfilingEnabled: () => !!activeSession || contains(process.execArgv, "--cpu-prof") || contains(process.execArgv, "--prof"),
             realpath,
-            debugMode: !!process.env.NODE_INSPECTOR_IPC || !!process.env.VSCODE_INSPECTOR_OPTIONS || some(process.execArgv, arg => /^--(inspect|debug)(-brk)?(=\d+)?$/i.test(arg)),
+            // debugMode: !!process.env.NODE_INSPECTOR_IPC || !!process.env.VSCODE_INSPECTOR_OPTIONS || some(process.execArgv, arg => /^--(inspect|debug)(-brk)?(=\d+)?$/i.test(arg)),
+            debugMode: true,

```

Comment:
replay.io 🥺

Reply:
Would really like to get #49027 if we think it's good FWIW

Reply:
Eh, it ought to happen after months of keeping this locally and trying my best to not commit this :P

<img src="https://media.giphy.com/media/xUPGcGFpLF6UCtJcd2/giphy.gif" alt="A woman saying 'okay, then you caught me'"/>